pom.xml

```diff
@@ -18,6 +18,8 @@
   <url>https://github.com/jenkinsci/plugin-util-api-plugin</url>
 
   <properties>
+    <jenkins.baseline>2.361</jenkins.baseline>
+    <jenkins.version>${jenkins.baseline}.2</jenkins.version>
```

Comment:
I understand that you want to use some JDK 11 features but these changes in the PR are not worth of such a huge user impact right now (maybe in a couple of weeks).